docs/docs/config/system.md

```diff
@@ -135,3 +135,40 @@ Following [split keyboard](../features/split-keyboards.md) settings are defined
 | `CONFIG_ZMK_SPLIT_BLE_PERIPHERAL_STACK_SIZE`            | int  | Stack size of the BLE split peripheral notify thread                       | 650                                        |
 | `CONFIG_ZMK_SPLIT_BLE_PERIPHERAL_PRIORITY`              | int  | Priority of the BLE split peripheral notify thread                         | 5                                          |
 | `CONFIG_ZMK_SPLIT_BLE_PERIPHERAL_POSITION_QUEUE_SIZE`   | int  | Max number of key state events to queue to send to the central             | 10                                         |
+
+## Snippets
+
+Snippets are a way to save common configuration separately when it applies to multiple different applications.
+
+More documentation is avaialable from [Zephyr](https://docs.zephyrproject.org/latest/build/snippets/index.html).
```

Comment:
```suggestion
More documentation is avaialable from [Zephyr](https://docs.zephyrproject.org/3.5.0/build/snippets/index.html).
```
We pin these and update on Zephyr upgrades so we don't have dead links in the future.

Reply:
Done.